frontend/amundsen_application/static/.betterer.results

```diff
@@ -455,9 +455,9 @@ exports[`eslint`] = {
       [19, 2, 8, "Assignment to function parameter \'resource\'.", "2131237679"],
       [20, 2, 248, "Expected a default case.", "1034339850"]
     ],
-    "js/ducks/tableMetadata/api/v0.ts:4044823741": [
-      [80, 8, 23, "Use object destructuring.", "1142306891"],
-      [139, 23, -4311, "Expected to return a value at the end of arrow function.", "5381"]
+    "js/ducks/tableMetadata/api/v0.ts:3883541609": [
```

Comment:
Changes in this file is auto generated by `npm run betterer:update`,
otherwise frontend test will fail in this pr.